src/Symfony/Bridge/PhpUnit/TextUI/Command.php

```diff
@@ -26,7 +27,7 @@ class Command extends BaseCommand
         /**
          * {@inheritdoc}
          */
-        protected function createRunner()
+        protected function createRunner(): BaseRunner
```

Comment:
@nicolas-grekas wait, aren't we already guaranteed to have php 7.0 here? We are inside a conditional block where phpunit is version 6 or above, *ergo* php itself must be 7 or above, correct? Or does the `class_exists('PHPUnit_Runner_Version')` have other implications?

Reply:
PHP 5.3 will still raise a parse error...

Reply:
Ah but 5.3 needs to be able to parse this I think.